packages/eas-cli/src/build/errors.ts

```diff
@@ -0,0 +1,53 @@
+export class EasCommandError extends Error {
+  constructor(message?: string) {
+    super(message ?? 'Unknown EAS error occurred.');
+  }
+}
+
+export class TurtleDeprecatedJobFormatError extends EasCommandError {
+  constructor(message?: string) {
```

Comment:
Make argument nonoptional in all those errors, it will avoid the need to provide a fallback string, plus it will be apparent that the server (or calling code) is a source of truth for error messages.